src/components/Button/Button.tsx

```diff
@@ -1,54 +1,24 @@
 import React from 'react';
 import styles from './Button.module.css';
 
-interface ButtonProps {
-  /**
-   * Is this the principal call to action on the page?
-   */
-  primary?: boolean;
-  /**
-   * What background color to use
-   */
-  backgroundColor?: string;
-  /**
-   * How large should the button be?
-   */
-  size?: 'small' | 'medium' | 'large';
-  /**
-   * Button contents
-   */
-  label: string;
-  /**
-   * Optional click handler
-   */
-  onClick?: () => void;
-}
+type ButtonElementProps = {
+  text: string;
+  color: string;
+  backgroundColor: string;
+  size: string;
+};
 
-/**
- * Primary UI component for user interaction
- */
-export const Button = ({
-  primary = false,
-  size = 'medium',
+export default function ButtonElement({
```

Comment:
I wouldn't call it `ButtonElement` it's a button, and you're creating components `Button` is sufficient